backend/pom.xml

```diff
@@ -14,6 +14,15 @@
     <logmanager.class>org.jboss.logmanager.LogManager</logmanager.class>
   </properties>
 
+  <repositories>
+    <repository>
+      <id>sonatype-oss-snapshots</id>
+      <name>Sonatype OSS SNAPSHOT Repository</name>
+      <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+      <!--// temporary repository in order to use Debezium SNAPSHOT nightly builds //-->
```

Comment:
Only temporary usage to get latest changes easily for the UI team without the need to build jars by themselves.

Reply:
Is this still needed at this point?

Reply:
I think as some time passed now :clock1030:, I have to check but quite certainly I can revert this now.